README.md

```diff
@@ -1,62 +1,58 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# COVID-19 tracking
 
-> Description the project.
+> A single page application thats displays COVID-19 information, the app shows the country name, flag active cases, number of deaths and total number recovered in a day.
 
+Get started by copying and pasting the following in your terminal:
 
-## Built With
+- 1
 
-- Major languages
-- Frameworks
-- Technologies used
+```
+git@github.com:Kayonga99/React-capstone-Covid-data.git
+```
 
-## Live Demo (if available)
+- 2
 
-[Live Demo Link](https://livedemo.com)
+```
+REACT-CAPSTONE-COVID-DATA
+```
 
+- 3
 
-## Getting Started
+```
+npm start
+```
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+```
+http://localhost:3000
+```
 
+- [click here! to see live demo](https://relaxed-kheer-37635b.netlify.app/)
 
-To get a local copy up and running follow these simple example steps.
+## Buit with
 
-### Prerequisites
+- REACT
+- REDUX
+- Webpack
+- CSS
 
-### Setup
 
-### Install
 
-### Usage
+## Author 
 
-### Run tests
+👤 **Kayonga Chiteta**
 
-### Deployment
+- GitHub: [@Kayonga99](https://github.com/Kayonga99)
+- Twitter: [@KayongaChiteta3](https://twitter.com/KayongaChiteta3)
+- LinkedIn: [Kkayonga Chiteta](https://www.linkedin.com/in/kayongac/)
 
 
+You can start editing the files in the "src" folder
 
-## Authors
+## Happy coding! 🌈
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
+if you are interested in learning more about webpack click the link below
 
 ## Show your support
```

Comment:
Great job @Kayonga99 🤝

- [ ] As the previous reviewer stated, kindly add the acknowledgment section in your readme file and give credit to Nelson Sakwa whose design was used.